source/request-as-event-emitter.ts

```diff
@@ -274,16 +226,15 @@ export default (options: NormalizedOptions, input?: TransformStream) => {
 			}
 		};
 
-		const {timeout} = options;
-		delete options.timeout;
-
-		if (options.dnsCache) {
-			options.lookup = options.dnsCache.lookup;
-		}
+		if (is.object(options.cache)) {
+			// `cacheable-request` doesn't support Node 10 API, fallback.
```

Comment:
Can you open an `cacheable-request` issue and link to it in this comment?

Reply:
I'm gonna rewrite `cacheable-request` completely and send a PR (it'll be a major version bump) :)